src/uucore/src/lib/features/sum.rs

```diff
@@ -368,8 +417,8 @@ impl_digest_common!(Sha3_512, 512);
 
 pub struct Shake128(sha3::Shake128);
 pub struct Shake256(sha3::Shake256);
-impl_digest_shake!(Shake128);
-impl_digest_shake!(Shake256);
+impl_digest_shake!(Shake128, 256);
+impl_digest_shake!(Shake256, 512);
```

Comment:
Are the values `256` and `512` correct? With the `impl_digest_common` macro, the values correspond to the bits in the algorithm name.